axum-macros/src/from_request.rs

```diff
@@ -313,11 +333,19 @@ fn extract_each_field_rejection(
                     }
                 }
             }
-        }
+        })
+    } else {
+        None
+    };
+
+    let impl_debug = if rejection_derive_opt_outs.derive_debug() {
+        Some(quote! { #[derive(Debug)] })
+    } else {
+        None
     };
```

Comment:
Here you don't even need `.transpose()?` 😉

```suggestion
    let impl_debug = rejection_derive_opt_outs.derive_debug().then(|| {
        quote! { #[derive(Debug)] }
    });
```

But of course the current code is fine too.

Reply:
Yeah thats definitely nicer!